ClimaArtifactsHelper.jl/src/ClimaArtifactsHelper.jl

```diff
@@ -205,7 +205,7 @@ function create_artifact_guided_one_file(
         Base.mv(downloaded_file, file_path)
     end
 
-    Base.cp(file_path, joinpath(output_dir, basename(file_path)))
+    run(`cp $file_path $(joinpath(output_dir, basename(file_path)))`)
```

Comment:
For what is worth, based on https://github.com/JuliaLang/julia/issues/30723#issuecomment-1738614721 this wouldn't work on Windows (but I don't know if you care about that platform)

Reply:
Thanks, @giordano

Reply:
I can confirm that `run(`cp $file_path $(joinpath(output_dir, basename(file_path)))`)` does not work on windows